src/main/java/com/softserve/sprintone/ivan/TaskIvanSprintOne.java

```diff
@@ -0,0 +1,422 @@
+        length = sc.nextFloat();
+        System.out.print("Type width of the cuboid: ");
+        width = sc.nextFloat();
+        System.out.print("type height of the cuboid: ");
+        height  = sc.nextFloat();
+        System.out.println("Volume of cuboid: " + getVolume(length, width, height));
+    }
+
+    /**
+     * Task 8 - Given a year, print the century it is in.
+     * @param sc Variable for receive data
+     */
+    public void task8(Scanner sc) {
+        int year, century;
+        System.out.print("Type a year: ");
+        year = sc.nextInt();
+        century = (int) Math.ceil(year / 100.0);
+        System.out.println("century = " + century);
+    }
+
+    /**
+     * Task 13 - The class Dinglemouse provided has a method plus100 which is supposed to return the number of the
+     * parameter incremented by 100. But it's not working properly.
+     * @param sc Variable for receive data
+     */
+    public void task13(Scanner sc) {
+        int number;
+        System.out.print("Type a number: ");
+        number = sc.nextInt();
+        System.out.println("Number incremented by 100 = " + Dinglemouse.dinglemouse.plus100(number));
+    }
+
+    /**
+     * Task 18 - Very simple, given an integer or a floating-point number, find its opposite.
+     * @param sc Variable for receive data
+     */
+    public void task18(Scanner sc) {
+        float number;
+
+        System.out.print("Type a number: ");
+        number = sc.nextInt();
+
+        if (number < 0)
+            System.out.println("Opposite number = " + -number);
+        else
+            System.out.println("Opposite number = " + -number);
+    }
+
+    /**
+     * This method returns a random element from a String List
+     * @param list A list of strings
+     * @return String
+     */
+    private String selectRandom(List<String> list){
+        return list.get((int) (random() * list.size()));
+    }
+
+    /**
+     * Task 23 - Rock, Paper, Scissors!
+     */
+    public void task23(){
+        final String player1, player2;
+        final List<String> rockPaperScissors = new ArrayList<>(){{
+                                                       add("rock");
+                                                       add("paper");
+                                                       add("scissors");
+                                                    }};
+
+        player1 = selectRandom(rockPaperScissors);
+        player2 = selectRandom(rockPaperScissors);
+
+        if (player1.equals(player2))
+            System.out.println("Player 1: " + player1 + "\nPlayer 2: " + player2 + "\nDraw!");
+
+        else if (player1.equals("scissors") && player2.equals("paper")
+                        || player1.equals("paper") && player2.equals("rock")
+                        || player1.equals("rock") && player2.equals("scissors"))
+            System.out.println("Player 1: " + player1 + "\n" + "Player 2: " + player2 + "\nPlayer 1 won!");
```

Comment:
Use formatting instead of concatenation (please review all the committed code and apply where applicable - not only in these lines)

take a look at https://www.baeldung.com/java-printstream-printf